cabal.project

```diff
@@ -59,6 +53,15 @@ package cardano-wallet-launcher
 package cardano-wallet-core-integration
   optimization: False
 
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/marlowe-cardano
+  tag: f5a0f64d8697dd7a29af91b11530039dc98a1d44
+  subdir:
+    marlowe
+    marlowe-symbolic
+    marlowe-actus
+
```

Comment:
If you have the patience, I might just delete all of the other `source-repository-package` entries and only add back in the ones needed to build.

Reply:
Yes, I wanted to do that as a second step